cmd/clusterctl/examples/openstack/provider-component/user-data/ubuntu/templates/worker-user-data.sh

```diff
@@ -86,10 +83,10 @@ EOF
 
 # Override network args to use kubenet instead of cni, override Kubelet DNS args and
 # add cloud provider args.
```

Comment:
Can you remove this instead of just commenting it out?

Reply:
@sbueringer  done